src/werkzeug/local.py

```diff
@@ -123,15 +139,22 @@ def _lookup():
 
         return LocalProxy(_lookup)
 
-    def push(self, obj):
+    def push(
```

Comment:
I think the types here should be `Any` (or `object`), but I'm not 100% sure.

Reply:
I think you are safe and correct with the `Any` annotation you've used.